uber/model_checks.py

```diff
@@ -196,12 +196,6 @@ def allowed_to_register(attendee):
 
 
 @validation.Attendee
-def printed_badge_deadline(attendee):
-    if attendee.is_new and attendee.has_personalized_badge and not c.SHIFT_CUSTOM_BADGES:
```

Comment:
FYI, removing this is going to cause a merge conflict with upstream. We'll deal with that though, it's a simple conflict to resolve.